src/main/java/frc/robot/commands/SwerveDriveWithController.java

```diff
@@ -81,4 +86,18 @@ public void end(boolean interrupted) {
     public boolean isFinished() {
         return false;
     }
+
+    // limiting joystick sense
+    public double limitedJoystickInput(double input) {
```

Comment:
```suggestion
    public double joystickResponseCurve(double input) {
```

id recommend changing the name to joystick response curve because its more clear what is happening

Reply:
Done.